test/semantics/canondo16.f90

```diff
@@ -15,13 +15,11 @@
 
 ! CHECK: A DO loop should terminate with an END DO or CONTINUE
 
-module iso_fortran_env
-  type :: team_type
-  end type
-end
+include '../../module/__fortran_builtins.f90'
+include '../../module/iso_c_binding.f90'
```

Comment:
Unfortunately, the same isn't true here. We should probably convert all of these to be run with `test_errors.sh`.